### PR TITLE
Locking items

### PR DESCRIPTION
**Pull request by lukasdt**
Added locking functionality to list items
Changed searchbar control
Added styles for checkbox

### CODE REVIEW

File: src/list-control/data.utils.js [hunk 1]
```diff
@@ -26,7 +14,23 @@ export default {
...
+        const x = { object: item, index: i };
+        lockedItems.push(x);
+      }
+    }
+    if (oldIndex < newIndex) {
+      lockedItems = lockedItems.reverse();
+    }
+    let data = arrayMove(items.toArray(), oldIndex, newIndex);
+    for (let i = 0; i < lockedItems.length; i += 1) {
+      const oi = data.indexOf(lockedItems[i].object);
```
Comment (elefantino) [inline src/list-control/data.utils.js]: Please use descriptive names for constants. What are oi and ni?

File: src/list-control/react-select-order-list.component.scss [hunk 2]
```diff
@@ -6,6 +6,7 @@ $oc-grid-font-size-icon:            1.8rem !default;
 // Colors
 $oc-grid-color-border:              #D3DADE !default;
 $oc-grid-color-text:                #67707C !default;
+$oc-grid-color-text-disabled:       #CCCCCC !default;
 $oc-grid-color-border:              #D3DADE !default;
```
Comment (elefantino) [inline src/list-control/react-select-order-list.component.scss]: Import colors from oc colors that you already have in styles/_colors

File: src/list-control/react-select-order-list.component.scss [hunk 3]
```diff
@@ -61,6 +67,12 @@ $oc-grid-color-border:              #D3DADE !default;
       fill: $oc-grid-color-text;
     }
   }
+  .oc-datagrid-selected-columns-locked-icon {
```
Comment (elefantino) [inline src/list-control/react-select-order-list.component.scss]: Please rename a 'datagrid' classname, it's not a datagrid component

File: examples/components/list-control/select-order-list.component.jsx [hunk 4]
```diff
@@ -12,24 +12,37 @@ export default class ListItemsView extends React.Component {
...
           value: 'three',
+          isLocked: false,
         },
       ]),
       selectedData: List([
+        {
+          label: 'one',
+          value: 'one',
+          isLocked: false,
+          isSelected: true,
```
Comment (Jen-ni) [inline examples/components/list-control/select-order-list.component.jsx]: selectedData contains only selected data, right? Therefore isSelected property is needeless. So I propose that isSelected is used only inside the component. isLocked should not be mandatory either.

**Pull request merged**